### ISSUE TITLE
Mod incompatibility: Graveyard Biomes and IronChest

### ISSUE THREAD

**Issue by SrRapero720** [hypherionmc/sdlink#47]
Installing this mod causes Server no start and throws incompatibility issues before start

 1: IRON CHEST
```log
[20:43:38] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: Found 27 dependencies adding them to mods collection
Exception in thread "main" java.lang.module.ResolutionException: Modules sdlink and com.fasterxml.jackson.core export package com.fasterxml.jackson.core to module ironchest
        at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
        at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
        at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
        at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
        at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
        at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:70)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
        at cpw.mods.bootstraplauncher@1.0.0/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149)
```

GRAVEYARD BIOMES
```log
[20:43:38] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: Found 27 dependencies adding them to mods collection
Exception in thread "main" java.lang.module.ResolutionException: Modules sdlink and com.fasterxml.jackson.core export package com.fasterxml.jackson.core to module ironchest
        at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
        at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
        at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
        at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
        at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
        at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:70)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
        at cpw.mods.bootstraplauncher@1.0.0/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149)
```

**Comment by SrRapero720** [comment]
Minecraft 1.18.2
Forge: 40.2.0
Mod: SimpleDiscordLink-forge-1.18.1-2.2.0.jar

**Comment by hypherionmc** [comment]
The incompatibility seems to be just with ironchest. Both errors refer to sdlink and ironchest being the conflict.


Problem with that is, just like with Minetogether, the "conflict" in question is correctly repacked on our side. For some reason, forge just wants to be an ass about it and complain like it does there.


We are working on our side to try and fix this specific conflict with any other mods that happen to rely on the same module. Hopefully we can find a solution soon

**Comment by hypherionmc** [comment]
Issue should be fixed in 2.2.3
